emission/core/wrapper/suggestion_sys.py

```diff
@@ -437,23 +438,32 @@ def review_start_loc_nominatim(lat, lon):
 11: "AIR_ON_HSR"
 '''
 
-def calculate_yelp_server_suggestion_singletrip_nominatim(tripid):
-    return_obj = { 'message': "Good job walking and biking! No suggestion to show.", 'method' : 'bike'}
-    start_lat, start_lon = geojson_to_lat_lon_separated(tripid["start_loc"])
-    end_lat, end_lon = geojson_to_lat_lon_separated(tripid["end_loc"])
+
+def calculate_yelp_server_suggestion_singletrip_nominatim(tripid, uuid):
+    all_users = pd.DataFrame(list(edb.get_uuid_db().find({}, {"uuid": 1, "_id": 0})))
+    user_id = all_users.iloc[all_users[all_users.uuid == uuid].index.tolist()[0]].uuid
+    timeseries = esta.TimeSeries.get_time_series(user_id)
+    cleaned_trips = timeseries.get_data_df("analysis/cleaned_trip", time_query = None)
+    spec_trip = cleaned_trips.iloc[cleaned_trips[cleaned_trips._id == tripid].index.tolist()[0]]
+    start_location = spec_trip.start_loc
+    end_location = spec_trip.end_loc
+    distance_in_miles = spec_trip.distance * 0.000621371
```

Comment:
nit: why do you need to convert the distance into miles? Just curious.

Reply:
The distance in miles is to compare with the calculated distances between the better suggested end point and original starting point.

Reply:
ok, but is there a reason you can't perform that comparison directly in `km`? Most scientific work is done in SI units.